doc/api/lxml.html.rst

```diff
@@ -0,0 +1,81 @@
+lxml.html
```

Comment:
Isn't there a way to generate these index files automatically? They seem trivial, and that's unlikely to change. But when new modules or packages are added, it's annoying to have to remember and add them here.

Reply:
There is `sphinx-autogen` [1] which can generate module rst files based on one simple top-level rst. You have to have summaries, which maybe is OK, but I can't see how to get the summaries to include private classes like lxml.etree._Element.

[1] https://www.sphinx-doc.org/en/master/man/sphinx-autogen.html

Reply:
Does it not take the default "autodoc" options into account?

Reply:
In the autodoc sections yes, but not in the autosummary tables



**Classes**


&nbsp; | &nbsp;
-- | --
AncestorsIterator(self, node[, tag]) | Iterates over the ancestors of an element (from parent to parent).
AttributeBasedElementClassLookup(self, …) | Checks an attribute of an Element and looks up the value in a class dictionary.
C14NWriterTarget | Canonicalization writer target for the XMLParser.
CDATA(data) | CDATA factory.
...

Reply:
Maybe it is possible to use `sphinx-apidoc` every time and forget about the summaries, it does take quite a few options, including `--private`.

Reply:
> Maybe it is possible to use `sphinx-apidoc` every time and forget about the summaries, it does take quite a few options, including `--private`.

Created #309 using this approach.